src/kern/npf.c

```diff
@@ -101,6 +101,7 @@ npf_destroy(npf_t *npf)
 	npf_conn_fini(npf);
 	npf_ifmap_fini(npf);
 
+	pserialize_unregister(npf->qsbr);
```

Comment:
@outscale-gle: Why did you add this call here?

If the thread which performed `npf_create()` processes the packets, then it must register itself using `npf_thread_register()` and, symetrically, remove itself using `npf_thread_unregister()` before calling `npf_destroy()`.